### PR TITLE
slang: Add version 0.26.0

### LINKED ISSUE #4603: [Request]: slang
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

slang

### Description

Slang is a new language and compiler for shader development from NVIDIA. It is designed to be a modern, extensible, and portable alternative to existing shading languages like HLSL and GLSL.

### Homepage

https://shader-slang.com/

### Download Link(s)

https://github.com/shader-slang/slang/releases

### Some Indication of Popularity/Repute

open sourced on Github with 1.1k star 👉 [shader-slang/slang](https://github.com/shader-slang/slang)

### PR DESCRIPTION
**Pull request by Locietta**
Closes #4603 

Upstreamed from my personal bucket: https://github.com/Locietta/sniffer/blob/master/bucket/slang.json

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/slang.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "0.26.0",
+    "description": "Slang is a new language and compiler for shader development. It is designed to be a modern, extensible, and portable alternative to existing shading languages like HLSL and GLSL.",
```
Comment (niheaven) [inline bucket/slang.json:3]:
```suggestion
    "description": "A new language and compiler for shader development.",
```

File: bucket/slang.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "env_add_path": "bin/windows-x86/release"
+        },
+        "arm64": {
+            "url": "https://github.com/shader-slang/slang/releases/download/v0.26.0/slang-0.26.0-win-arm64.zip",
+            "hash": "7d558b6f0f1cd095e97c12ab3da0f96f18e5e5e05abb25831095e37a405ed8cf",
+            "env_add_path": "bin/windows-aarch64/release"
+        }
+    },
+    "extract_dir": "",
+    "bin": "",
```
Comment (niheaven) [inline bucket/slang.json:24]:
```suggestion
```

These are not needed.

**Review by niheaven — CHANGES_REQUESTED** [review]

**Review by niheaven — APPROVED** [review]

**Comment by rashil2000** [comment]
Is adding to Path really required? We could just make shims for `slangc.exe` and `slangd.exe` using the bin field.

**Pull request merged**